docs/swagger.yaml

```diff
@@ -204,96 +204,183 @@ components:
+          nullable: true
+        unitPrice:
+          type: string
       required:
-        - amountExcludingTax
-        - amountIncludingTax
-        - description
-        - id
-        - quantity
-        - taxAmount
+        - totalAmount
+        - totalAmountExcludingTax
+        - totalTaxAmount
         - taxPercentage
+        - currency
+        - name
+        - id
+        - unitInfo
         - discount
         - quantityUnit
-    Receipt:
-      title: Receipt
+    BottomLine:
+      title: BottomLine
       type: object
+      description: Summary of the order
       properties:
-        orderLines:
-          type: array
-          minItems: 1
-          items:
-            $ref: '#/components/schemas/OrderLine'
+        totalAmount:
+          type: string
+        totalTax:
+          type: string
+        totalDiscount:
+          type: string
+        currency:
+          $ref: '#/components/schemas/CurrencyEnum'
         shippingAmount:
           type: integer
+          description: Shipping amount for the order
+          nullable: true
+        tipAmount:
+          type: integer
+          description: Tip amount for the order
           nullable: true
+        terminalId:
+          type: string
+        recycleDiscount:
+          type: string
+          description: 'Discount due to recycling '
+        coupon:
+          type: string
+          description: Amount paid by gift card or coupon
       required:
-        - orderLines
-        - shippingAmount
-    ImageId:
-      title: ImageId
+        - totalAmount
+        - totalTax
+        - totalDiscount
+        - currency
+    UnitInfo:
+      title: UnitInfo
       type: object
+      description: Optional. If no quantity info is provided the order line will default to 1 pcs
+      x-examples:
+        example-1:
+          value:
+            quantity: '0.822'
+            unitPrice: 15000
+            quantityUnit: kg
+        example-2:
+          value:
+            quantity: '2'
+            unitPrice: 20000
+            quantityUnit: pcs
       properties:
-        id:
+        unitPrice:
+          type: integer
+          description: Price per unit
+          nullable: true
+        quantity:
+          type: string
+          maxLength: 10
+          pattern: '^\d+([\.]\d{1,8})?$'
+          example: '0.822'
+          description: 'Quantity given as a integer or fraction '
+        quantityUnit:
+          $ref: '#/components/schemas/QuantityUnitEnum'
+        '':
           type: string
-          pattern: '^[0-9A-Za-z-_\.]+$'
-          maxLength: 128
       required:
-        - id
-    CategoryEnum:
+        - unitPrice
+        - quantity
+        - quantityUnit
+    CurrencyEnum:
       type: string
-      title: CategoryEnum
+      title: CurrencyEnum
       enum:
-        - general
-        - receipt
-        - orderConfirmation
-        - delivery
-        - ticket
-        - booking
+        - EUR
+        - NOK
```

Comment:
only NOK support today, expand when our app and payments platform supports others

Reply:
🤷 . For Order management currency will only be ux in the begging, but doesnt matter much either way. I can remove it